Sources/DittoDiskUsage/Constants.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  File.swift
```

Comment:
nit: Maybe I'm not familiar with Swift/Xcode, but should all these be the name of the file as opposed to just File.swift?

Reply:
Good catch - Xcode defaults every new file to `File.swift` and I missed fixing these

Reply:
Updated here and in every other file that had the same old placeholder